src/write_outputs/write_charge.jl

```diff
@@ -4,7 +4,8 @@
 Function for writing the charging energy values of the different storage technologies.
 """
 function write_charge(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
-    gen = inputs["RESOURCES"]
+    gen = inputs["RESOURCES"]   # Resources (objects) 
+    resources = inputs["RESOURCE_NAMES"]    # Resource names
```

Comment:
Perhaps we should work to make the interface with `inputs` "smaller" by just doing `resources = resource_name.(gen)`